caper/caper/views.py

```diff
@@ -422,16 +453,89 @@ def project_page(request, project_name, message=''):
     dfl = []
     for _, dlist in samples.items():
         dfl.append(pd.DataFrame(dlist))
-
     aggregate = pd.concat(dfl)
+    aggregate.columns = [col.replace(' ', "_") for col in aggregate.columns]
+    proj_id = str(project['_id'])
```

Comment:
This breaks project creation as the `_id` is somehow not set for a new project when this line is called. I am attempting to fix.

Reply:
In some parts of the code we assume `_id` exists, and in others we assume it does not necessarily exist. This current code doesn't seem to break anything I suppose, so let us leave it until we find it breaks something.